reactioncommerce/reaction-next-starterkit
# feat: 12 nnnnat product grid layout

issues: #12 #13 #15 
impact: **minor**
type **feat**

## Issue
Storefront app needed UI and Routing created for the Products Grid

## Solution
Created two new components ProductGrid and ProductItem using material-ui.
Included @willopez  routing updates.

## Testing
1. Spin up the Storefront app and see a grid of Products with titles, vendors, prices, images and status badges.
2. Click either a product title or image to navigate to the product's detail page

## Notes
*  Product catalog data is being mocked from a file called `tempProducts.js`
*  Product images are being mocked from [placehold.it](http://placehold.it)
*  Seeing a server error when running inside of docker `TypeError: Cannot set property 'networkStatus' of undefined` shouldn't be related to any work form this branch
